.github/workflows/release.yml

```diff
@@ -251,7 +255,7 @@ jobs:
           git config --global user.name "edgelessci"
           git config --global user.email "edgelessci@users.noreply.github.com"
           git tag -a "${VERSION}" -m "Release ${VERSION}"
-          git push origin "refs/tags/${VERSION}"
+          git push --force origin "refs/tags/${VERSION}" # force push to overwrite existing tag
```

Comment:
This force push should never be needed, please remove.

Reply:
Let's move the tagging to the on-release workflow in a separate PR. We can allow the force for now.